src/Template/Events/add.ctp

```diff
@@ -320,7 +320,13 @@
                     ]) ?>
                     <?= $this->Form->input('tools._ids', [
                         'options' => $tools,
-                        'help' => 'If this event or class requires certain tools to be available for use during then select any that apply.',
+                        'help' => [
+                            $this->Form->button('Clear tool selection', [
+                                'type' => 'button',
+                                'id' => 'clear-tools',
+                            ]),
+                            '&nbsp;If this event or class requires certain tools to be available for use during then select any that apply.',
```

Comment:
Works fine in testing however I'd like to suggest we add a <BR> here at the beginning of the line so that the checkbox is clearly on its own.

Reply:
Sorry about the hidden BR, my bad.  Yes, that was my suggestion, so it shows up more clearly.

Reply:
btw @excalbian ^ this was what I was talking about in my comment. 
I'm going to go ahead and and add a commit to clean up this a tad bit, and then get everything in line with history and give it a thumbs up